src/gpt_fn/completion.py

```diff
@@ -63,6 +63,51 @@ def function_completion(
     )
 
 
+async def function_acompletion(
```

Comment:
覺得 a 放在最前面好像比較好找一點~!
```suggestion
async def afunction_completion(
```